pages/projects/_projectId.vue

```diff
@@ -0,0 +1,178 @@
+<template>
+  <div class="project-details">
+    <div class="project-details__page-route">
+      <p>
+        <nuxt-link
+          :to="{
+            name: 'data',
+            query: {
+              type: 'sparcAward'
+            }
+          }"
+        >
+          Teams and Projects
+        </nuxt-link>
+        > {{ formatBreadcrumb(fields.title) }}
+      </p>
+    </div>
+    <details-header>
```

Comment:
There is no `.container` wrapping this body content to maintain our "max width". See other pages for examples of this.